exporter/sematextexporter/testdata/config.yaml

```diff
@@ -0,0 +1,18 @@
+sematext/default-config:
+sematext/override-config:
+  endpoint: http://localhost:8080
+  timeout: 500ms
+  sending_queue:
+    enabled: true
+    num_consumers: 3
+    queue_size: 10
+  retry_on_failure:
+    enabled: true
+    initial_interval: 1s
+    max_interval: 3s
+    max_elapsed_time: 10s
+  region: US  
+  app_token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+  metrics_schema: otel-v1
+  payload_max_lines: 72
+  payload_max_bytes: 27
```

Comment:
Should you validate this and other values above? Could one put -69 here? What would that do?

Reply:
This is just testdata, does not really affect the exporter

If check the config_test file you will see where it was implemented